src/astarte-client/client.ts

```diff
@@ -407,25 +409,43 @@ class AstarteClient {
     await this.$delete(this.apiConfig.flowInstance({ ...this.config, instanceName: flowName }));
   }
 
-  async getPipelineDefinitions(): Promise<any> {
+  private async getPipelineNames(): Promise<Array<AstartePipeline['name']>> {
```

Comment:
Because it is just used in `getPipelines()` inside this class. So I left it `private`, postponing the decision of which API the client should expose for a later time

Reply:
The logic so far has been to map all the REST API options, plus some utility functions.
Also, the getPipelines is way more expensive (1 api call + 1 additional call for every name returned)

Reply:
Removed the `private` marker